opm/simulators/linalg/ISTLSolverEbos.hpp

```diff
@@ -384,9 +388,6 @@ class WellModelMatrixAdapter : public Dune::AssembledLinearOperator<M,X,Y>
             SPPointer sp(ScalarProductChooser::construct(parallelInformation_arg));
 #endif
 
-            // Communicate if parallel.
-            parallelInformation_arg.copyOwnerToAll(istlb, istlb);
```

Comment:
I am missing the overview. Is this part ILU0 specific or do we get here if we use other preconditioners, too?